src/coreComponents/codingUtilities/StringUtilities.hpp

```diff
@@ -180,8 +180,8 @@ string_view trimSpaces( string_view str );
  * @param[in] strToRemove the string to search for in the line
  * @return the new (truncated) string
  */
-string removeStringAndFollowingContent( string const & str,
-                                        string const & strToRemove );
+string_view removeStringAndFollowingContent( string_view str,
+                                             string_view strToRemove );
```

Comment:
Please update the doc by adding that we get a `view` onto the subpart of interest of `str`.
So if you modify `str` then the returned result will be modified as well.
In case that's an issue, then we should copy.